src/Controls/src/Core/Handlers/Toolbar/ToolbarHandler.Android.cs

```diff
@@ -0,0 +1,398 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Android.Content;
+using Android.Content.Res;
+using Android.Graphics;
+using Android.Graphics.Drawables;
+using Android.Views;
+using AndroidX.AppCompat.App;
+using AndroidX.AppCompat.Graphics.Drawable;
+using AndroidX.DrawerLayout.Widget;
+using Google.Android.Material.AppBar;
+using Microsoft.Maui.Controls.Platform;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Handlers;
+
+namespace Microsoft.Maui.Controls.Handlers
+{
+	internal class ToolbarHandler : ElementHandler<Toolbar, MaterialToolbar>
```

Comment:
All the commented out code in here is related to FlyoutPage and once we wire that all up